client/src/pages/Login/Login.tsx

```diff
@@ -24,16 +36,34 @@ export default function Login(): JSX.Element {
         }
       }
     } catch (err) {
+
+      // Log any errors to the console
       console.log(err);
     }
   };
 
+  // Render login form
   return (
-    <div className={styles['Login']}>
-      <p>Login Page</p>
-      <TextField header="Username or Email" setInputValue={setUserInput} />
-      <PasswordField header="Password" setInputValue={setPassword} />
-      <Button onClick={handleLogin}>Login</Button>
+    <div><p>Login Page</p>
+      <div className={styles['Login__LoginContainer']}>
+        <div className={styles['Login__whiteContainer']}>
+          <div className={styles['Login__logo']}> <img src = {logo} alt = {"Roadwatch Logo"}/></div>
```

Comment:
Can you put the img tag on a next line and have the closing div tag be below? instead of all on one line. If prettier formats it this way its fine otherwise I think it would be nicer not on 1 line

Reply:
I put it on a new line. Thanks!